install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+ack-4.12-kube-1.26-api-removals-in-4.13: |-
+    Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
```

Comment:
Comparing with the 4.11-to-4.12 note, we want an extra two spaces of indent here too:

```console
$ git --no-pager diff origin/release-4.11..origin/pr/908 -- install/0000_00_cluster-version-operator_01_admingate_configmap.yaml
diff --git a/install/0000_00_cluster-version-operator_01_admingate_configmap.yaml b/install/0000_00_cluster-version-operator_01_admingate_configmap.yaml
index 6d897cfe..405f0e03 100644
--- a/install/0000_00_cluster-version-operator_01_admingate_configmap.yaml
+++ b/install/0000_00_cluster-version-operator_01_admingate_configmap.yaml
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
 data:
-  ack-4.11-kube-1.25-api-removals-in-4.12: |-
-      Kubernetes 1.25 and therefore OpenShift 4.12 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6955381 for details and instructions.
+ack-4.12-kube-1.26-api-removals-in-4.13: |-
+    Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
 metadata:
   name: admin-gates
   namespace: openshift-config-managed
 ```